api/src/pcapi/routes/backoffice_v3/templates/accounts/get/details/bookings.html

```diff
@@ -0,0 +1,67 @@
+{% if bookings %}
+  <table class="table mb-4">
+    <thead>
+      <tr>
+        <th scope="col"></th>
+        <th scope="col">Offreur</th>
+        <th scope="col">Nom de l'offre</th>
+        <th scope="col">Prix</th>
+        <th scope="col">Date de résa</th>
+        <th scope="col">État</th>
+        <th scope="col">Contremarque</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for booking in bookings %}
+        <tr>
+          <th scope="row" data-bs-toggle="collapse" data-bs-target="#b{{ booking.id }}">
+            <i class="bi bi-chevron-down"></i>
+          </th>
+          <td>{{ booking.offerer.name | escape }}</td>
+          <td>
+            <a href="{{ booking.stock.offer | pc_pro_offer_link }}" target=blank_" style="z-index: 999;">
```

Comment:
(détail) Si on veut un nouveau onglet/fenêtre, c'est `_blank` (et il manque un guillemet ouvrant) : 

```suggestion
            <a href="{{ booking.stock.offer | pc_pro_offer_link }}" target="_blank" style="z-index: 999;">
```

Le z-index me paraît bizarre, mais il y a sûrement une bonne raison ?

Reply:
Oups, il a dû se passer un "truc". `z-index` était un essai pour un problème quand on testait la classe `table-hover` sur la table ; effectivement sans intérêt. Erreurs corrigées.